server/services/gameplay.go

```diff
@@ -26,26 +26,61 @@ func convertMoveRequestInward(req *pb.MoveRequest) *controller.MoveRequest {
 	return domMoveRequest
 }
 
-func convertMoveResponseOutward(res *controller.MoveResponse) *pb.MoveResponse {
+func convertMoveResponseOutward(res controller.MoveResponse) *pb.MoveResponse {
 	pbMoveResponse := pb.MoveResponse{}
 	pbMoveResponse.MoveSuccess = res.MoveSuccess
 	pbMoveResponse.Message = &res.Message
 
 	return &pbMoveResponse
 }
 
+func convertBoardSubscriptionRequestInward(req *pb.BoardSubscriptionRequest) controller.BoardSubscriptionRequest {
+	domBoardSubscriptionRequest := controller.BoardSubscriptionRequest{
+		GameID: req.GetGameId(),
+	}
+
+	return domBoardSubscriptionRequest
+}
+
+func convertBoardUpdateOutward(res controller.BoardUpdate) *pb.BoardUpdate {
+	pbBoardUpdate := pb.BoardUpdate{}
+
+	pbBoard := pb.BoardState{
+		Board: make([]pb.Tile, 32),
```

Comment:
Magic number, can we define a constant out-of-line?

Reply:
Defined in controller